pkg/containertools/runner.go

```diff
@@ -101,6 +102,49 @@ func (r *ContainerCommandRunner) Save(image, tarFile string) error {
 	return nil
 }
 
+// Unpack copies a directory from a local container image to a directory in the local filesystem.
+func (r *ContainerCommandRunner) Unpack(image, src, dst string) error {
```

Comment:
Shouldn't this replace `Save` entirely? I don't think we want to use save at all, unless I am missing a case where we want it.

Reply:
Good point; it should. I'll drop `Save` in favor of `Unpack`.